tests/components/samsungtv/test_media_player.py

```diff
@@ -297,10 +297,7 @@ async def test_update_connection_failure(hass, remotews, mock_now):
     ):
         await setup_samsungtv(hass, MOCK_CONFIGWS)
 
-        with patch(
-            "homeassistant.components.samsungtv.bridge.SamsungTVWS",
-            side_effect=ConnectionFailure("Boom"),
-        ):
+        with patch.object(remotews, "open", side_effect=ConnectionFailure("Boom")):
```

Comment:
This moves the side_effect to the correct location, as the constructor cannot raise `ConnectionFailure`, only the `open` method.